VATSpy.dat

```diff
@@ -18368,12 +18369,13 @@ ZHHH|Wuhan ACC (Cover ZHHH 7800m and Below) - Wuhan|ZHHH|
 ZHWH|Wuhan CTR (Cover ZHCC and ZHHH 7800m and Below) - Wuhan|ZHWH|
 ZGHA|Changsha ACC (Cover ZGHA 7800m and Below [ZGGG Overlap area 7800m Above]) - Guangzhou|ZGHA|
 ZGNN|Nanning ACC - Guangzhou|ZGNN|
+ZGNN-AR07|Nanning ACC AR07 (Cover 7800m and Below [ZGGG Overlap area 7800m Above]) - Guangzhou|ZGNN|
 ZGNN-AR08|Nanning ACC AR08/09 (Cover 7800m and Below [ZGGG Overlap area 7800m Above]) - Guangzhou|ZGNN|
-ZGZJ|Zhanjiang ACC [AR01/03(Cover 7800m and Below)][AR02(Cover 8900m and Below)] - Guangzhou|ZGZJ|
+ZGNN-AR11|Nanning ACC [AR10/12(Cover 7800m and Below)][AR11(Cover 8900m and Below)][ZJSA Overlap area Above] - Guangzhou|ZGNN|

```

Comment:
We have been accepting these before, but it is against the purpose of the dataset. We therefore can not accept new contributions such as this. 
Names such as this are very long, confusing and cluttering the dataset.

The dataset does not support level splits and other regions such as Germany has already been rejected doing stuff such as this. You will have to make a choice of either showing what the airspace is enroute or what the airspace covers on lower levels.

Reply:
This is different from the situation in Germany, where they are both enroute sector. The additional display was added to let the pilot know which ACC on his/her altitude belonged to and had no practical significance. But I can do it to simplify it.

Reply:
The data gets filled with unnessecerily information, with "duplicate" sectors which clutters the map.

Looking at the excisting data, this is just not something we want, it is way to excessive.
![image](https://user-images.githubusercontent.com/35482228/161004988-d286030f-cf77-4679-bc20-56e0310252cc.png)

> This is different from the situation in Germany, where they are both enroute sector. 
Confused what you mean here. The situation in Germany I am talking about is an area above Innsbruck where LOWW covers a lower part and EDDM covers and upper part. They too attemtped to add "fake" sectors with sector information like this.


Is is upto the controller to know where a sector stops or begins, and handover the pilots accordingly. The controller info to the controller can be utalized to describe level splits if you want to, but our dataset does not serve this purpose.

Reply:
I see what you mean, you wish I could remove all the fake sectors?

Reply:
@Adrian2k done please check it